pkg/webhook/resources/virtualmachine/validator.go

```diff
@@ -540,6 +540,13 @@ func (v *vmValidator) checkReservedMemoryAnnotation(vm *kubevirtv1.VirtualMachin
 	if reservedMemory.Cmp(*mem) >= 0 {
 		return werror.NewInvalidError("reservedMemory cannot be equal or greater than limits.memory", field)
 	}
+	if vm.Spec.Template.Spec.Domain.Memory != nil && vm.Spec.Template.Spec.Domain.Memory.Guest != nil {
+		totalMemory := *vm.Spec.Template.Spec.Domain.Memory.Guest
+		totalMemory.Add(reservedMemory)
+		if totalMemory.Cmp(*mem) > 0 {
+			return werror.NewInvalidError("guest memory + reservedMemory cannot be greater than limits.memory", field)
```

Comment:
can we include the actual values of the guest, reserved and limit in the error message?